src/Disks/DiskLocal.h

```diff
@@ -101,7 +101,8 @@ class DiskLocal : public IDisk
 
     void truncateFile(const String & path, size_t size) override;
 
-    DiskType getType() const override { return DiskType::Local; }
+    DataSourceDescription getDataSourceDescription() const override;
```

Comment:
why not like this
```suggestion
    const DataSourceDescription & getDataSourceDescription() const override;
```

Reply:
I don't want. Maybe unsuitable somewhere.